README.md

```diff
@@ -94,9 +104,20 @@ deactivate
 module purge
 ```
 
-We highlight that the module files need to be loaded before every activation
+Please note the module files need to be loaded before every activation
 of the virtual environment.
 
+NumPy 2 users need to override the version requirements as follows:
+
+```sh
+sed -i "/numpy/d" requirements.txt
+python3 -m pip install -r requirements.txt "numpy==2.1" "pandas>=2"
```

Comment:
This sed expression is unfortunate, but I couldn't think of a better alternative. There is a proposal to introduce a pip flag [`--ignore-conflicts`](https://pip.pypa.io/en/latest/ux-research-design/research-results/override-conflicting-dependencies/#recommendations) that would allow users to override version requirements from `requirements.txt` by specifying new requirements as command line arguments. A draft implementation was proposed, but it's currently stalled due to the complexity of gracefully handling edge cases where no suitable version of a package can be determined due to version requirements of indirect dependencies not being satisfied. Providing two different requirement files goes against pip best practices. Forcing users to migrate to NumPy 2 is also not an option, since cluster users must use NumPy 1 when loading modules from EasyBuild or EESSI.

Reply:
Maybe we could do the following:
```sh
# for NumPy 1 users
python3 -m pip install -r requirements.txt "numpy<2.0" "pandas<2.0"
```
```sh
# for NumPy 2 users
python3 -m pip install -r requirements.txt "numpy>=2.1" "pandas>=2.0"
```

This way both user communities have a similar-looking pip install command, and we can remove the `<2` restriction from `requirements.txt`. This would also better reflect how pyMBE supports both NumPy releases.

Reply:
@jngrad I think that I like this last suggestion more. Using sed can be a bit "blackboxy" for people not so familiar with bash commands, which could potentially confuse some of our users.

Reply:
Done!